content/docs/hello-world.md

```diff
@@ -1,12 +1,12 @@
 ---
 id: hello-world
-title: Hello World
+title: Kamusta Mundo
```

Comment:
```suggestion
title: Hello World
```

I think we retain ```Hello World``` just to be consistent with the codepen snippet

Reply:
@elrmndo Pwede ba nating palitan lahat ng `Hello, world!` ng `Kamusta, mundo!`? Was looking at other languages and binago nila yun in their language https://fr.reactjs.org/docs/introducing-jsx.html, https://pl.reactjs.org/docs/introducing-jsx.html, https://ru.reactjs.org/docs/introducing-jsx.html. It's just a nice touch imo.